### DISCUSSION TITLE
How do I connect between SQL server and budibase?

### DISCUSSION THREAD

**Discussion by LTCTHITA** [Budibase/budibase#12975] [Help]
Hi,

I have a SQL server with the SQL server management studio program. But I don't know how to connect to Budibase.
I don't know where to get this information.

![Screenshot 2024-02-07 105535](https://github.com/Budibase/budibase/assets/93240913/fed0cd70-4213-4479-88c2-a832139f4d0c)

thank you,

**Comment by ConorWebb96** [comment]
Heres the docs for ms-sql-server. Hopefully this helps. https://docs.budibase.com/docs/ms-sql-server
